### PR TITLE
add find by id query example

### PR DESCRIPTION
**Pull request by nlandolfi**

### CODE REVIEW

**Comment by nlandolfi** [comment]
want to slowly build up a small collection of examples

File: jobs/graphql/README.md [hunk 1]
```diff
@@ -32,6 +32,31 @@ query {
...
 }
+```
+Suppose you have a task's id,  `<the_id>`, you can query for it specifically by:
+```
+{
+  records(query: {
+    kind: TASK, 
+    filters: [
+        {
+            op: EQ, f
```
Comment (viggyfresh) [inline jobs/graphql/README.md]: extra f here

File: jobs/graphql/README.md [hunk 2]
```diff
@@ -32,6 +32,31 @@ query {
...
+            ield: "id",
```
Comment (viggyfresh) [inline jobs/graphql/README.md]: f should be here!

File: jobs/graphql/README.md (hunk 1 above)
Comment (nlandolfi) [reply jobs/graphql/README.md]: lol

**Pull request merged**